js/ui/map.js

```diff
@@ -586,7 +594,7 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
         } else if (style instanceof Style) {
             this.style = style;
         } else {
-            this.style = new Style(style, this.animationLoop);
+            this.style = new Style(style, this.animationLoop, this._workerCount);
```

Comment:
@lucaswoj As long as `Map` has a `setStyle` method, I think it will need to keep a copy like this of the `workerCount`, because it needs to be able to do `new Style(...)` not just in the map's constructor.  We _could_ add `workerCount` as a parameter to `setStyle`, but that seems a little odd to me.